### DISCUSSION TITLE
DataTable not stacking for smaller screen size

### DISCUSSION THREAD

**Discussion by chrishj59** [primefaces/community#205] [PrimeReact] [answered]
On the v10 documentation site [https://primereact.org/datatable/#row_selection_events](url), if use the responsive design mode in Firefox (v118.0.1) and use the iphone11 vertical layout only the 1st columns are visible.

I seem to remember that the JSF  version of PrimeFaces had the table switching to vertical stacking.

Is it possible to have something similar for the PrimeReact version?

I have done a search and on the PrimeFaces version seems to come up in the search results.

Many thanks,

Chris.

**Comment by cagataycivici** [comment]
```jsx
<DataTable responsiveLayout="stack"
```

should help.

**Reply by chrishj59** [reply, marked as answer]
This works great on the documentation pages.
Many thanks

**Reply by chrishj59** [reply]
I notice that it is marked as @deprecated since version 9.2.0 is there a replacement on v10?

**Reply by Ctrl-Zee** [reply]
I noticed that as well. Is there a better way to implement the stacked responsive layout that won't break in the future?
